### PR TITLE
[TT-1806] add required chain.link labels to k8s soak test

### PR DESCRIPTION
**Pull request by Tofel**
**Why?**
To better track k8s expenses

### CODE REVIEW

**Review by Farber98 — APPROVED** [review]

File: integration-tests/common/common.go [hunk 1]
```diff
@@ -384,10 +384,24 @@ func BuildNodeContractPairID(node *client.ChainlinkClient, ocr2Addr string) (str
 }
 
 func (c *Common) Default(t *testing.T, namespacePrefix string) (*Common, error) {
+	productName := "data-feedsv2.0"
```
Comment (krebernisak) [inline integration-tests/common/common.go:387]: Is this the right product name?

**Review by krebernisak — APPROVED** [review]

File: integration-tests/common/common.go [hunk 2]
```diff
@@ -384,10 +384,24 @@ func BuildNodeContractPairID(node *client.ChainlinkClient, ocr2Addr string) (str
...
+	nsLabels, err := environment.GetRequiredChainLinkNamespaceLabels(productName, "soak")
+	if err != nil {
+		return nil, err
+	}
+
+	workloadPodLabels, err := environment.GetRequiredChainLinkWorkloadAndPodLabels(productName, "soak")
```
Comment (aalu1418) [inline integration-tests/common/common.go:393]: do we need to differentiate between `soak` vs `smoke`?

**Review by aalu1418 — APPROVED** [review]

File: integration-tests/common/common.go (hunk 1 above)
Comment (Tofel) [reply integration-tests/common/common.go:387]: it's OCRv2 test => `"data-feedsv2.0"`, it doesn't matter whether it's running on Solana or Ethereum or some other chain.

File: integration-tests/common/common.go (hunk 2 above)
Comment (Tofel) [reply integration-tests/common/common.go:393]: good callout, we actually should

Comment (Tofel) [reply integration-tests/common/common.go:393]: I think in this case these tests are always `soak`, no?

**Review by chainchad — APPROVED** [review]

**Pull request merged**